benchmarks/eko_bench.py

```diff
@@ -0,0 +1,126 @@
+# -*- coding: utf-8 -*-
```

Comment:
Ok, I thought you were using this really in a "broader sense" - but since this is strictly related to the LHA setting, please merge this file file `lha_...` e.g. you can extend `BenchmarkVFNS` to allow other externals ... otherwise there is too much code repeated

Reply:
okay I can do it!